### ISSUE TITLE
Textual description of Association underrepresents FMI

### ISSUE THREAD

**Issue by dzimmer** [modelica/www.modelica.org#91]
https://modelica.org/association/
The textual description needs an update to take FMI into account

**Comment by t-sommer** [comment]
FMI is already in the list of MA projects. What exactly are you missing?

**Comment by casella** [comment]
> > The Modelica Association is a non-profit, non-governmental organization with the aim of developing and promoting the Modelica modeling language for modelling, simulation and programming of physical and technical systems and processes.

This should have been changed after the 2017 Strategy Meeting, when we re-focused the MA from being centered on the Modelica language to a multi-project umbrella organization. I remember we came up with a mission statement such as "Developing and promoting open and coordinated standards for system modelling and simulation". I'm not sure we actually wrote this somewhere, but of course this should show up in the home page for sure.

> It is a minor thing. I'll work on a proposal to improve this.

It's not a minor thing, conveying what is the core mission of the association is a key point.

**Comment by chrbertsch** [comment]
> I remember we came up with a mission statement such as "Developing and promoting open and coordinated standards for system modelling and simulation". I'm not sure we actually wrote this somewhere, but of course this should show up in the home page for sure.

A similar sentence is part of the MA bylaws (https://github.com/modelica/MA-Bylaws/blob/master/MA-Bylaws.md#-2-purpose-of-the-association:
_"MA is a non-profit, non-governmental organization. The scope of business comprises coordinated standardization and development of software technology and methods in the area of cyber physical- systems and systems engineering. In particular, MA develops and promotes the Modelica language and Modelica libraries, for specification, modeling, simulation, other analyses and design methods, as well as systems operations of physical and technical systems and processes."_
In the "In particualar ..." we could also widen the scope beyond Modelica language and libraries ...

**Comment by dzimmer** [comment]
I had a discussion with Hubetus yesterday and he reminded me as well about the 2017 strategy meeting. I will look for a fitting statement and may change the corresponding PR #107 accordingly.

**Comment by casella** [comment]
The mission statement I quoted above

> Developing and promoting open and coordinated standards for system modelling and simulation

doesn't contain the word "Modelica" on purpose, to avoid conveying the implicit message that the MA is primarily about the Modelica language. It used to be like that, but it's changed since the MA became an umbrella association in 2012, and even more so after the 2017 strategy meeting, which really put this fact into focus. Unfortunately, we've been really slow in reacting to that in terms of communication, but it's never too late. Since then, there's been a lot of non-Modelica language developments, new standards and MAPs, so it's even more true.

I checked the [minutes](https://github.com/modelica/MA-Internal/blob/2b865d30a39d3296b267050d2cbc71c7e5a6ff88/Meetings/2017-10-17-Strategy-Lund/2017-10-17-MA_strategy_minutes.docx) of the strategy meeting, which make a good reading. I could not find such a statement explicitly, but I remember that @DagBruck formulated it as such at some point. For sure we wrote it down somewhere, but I can't find where.

I searched the minutes for all instances of the "open" keyword and I found the following statements about the MA, which all point in that direction.

- Globally recognized organization developing open coordinated standards for seamless work flows in model based engineering (design, test, operation, diagnostics, optimization, ...)
- "Open standard" is a unique selling point.
- MA provides innovative solutions, in an open eco-system.
- The results of MA, standard texts + software, are open source under a permissive license and can be downloaded and used without fees (_this is a nontrivial point, other organizations sell their standard documents at very high prices_)
- Open + innovative seems to be specific to MA.
- Provide innovative solutions, in an open eco-system
- Vision in 10 years (2027): Globally recognized organization developing open coordinated standards for seamless work flows in model based engineering (design, test, operation, diagnostics, optimization, ...)

Food for thought 😃
